packages/geocoder/src/geocoders/here.ts

```diff
@@ -73,8 +73,9 @@ export default class HereGeocoder extends Geocoder {
     const { items } = response;
     return {
       features: items
-      // Here can return continued query suggestions, which we do not support.
-      ?.filter(item => item.resultType !== "chainQuery")
+      // Here has various types of responses, some of which are not locations.
+      // We only want the actual places, so throw out any without a position.
+      ?.filter(item => item.position !== undefined)
```

Comment:
```suggestion
      ?.filter(item => !!item.position)
```
Does this also work?

Reply:
Sure, I was being unnecessarily verbose.